pkg/codegen/internal/test/testdata/simple-enum-schema/types/enums.ts

```diff
@@ -0,0 +1,25 @@
+// *** WARNING: this file was generated by test. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+import * as pulumi from "@pulumi/pulumi";
+
+export const redContainerColor: ContainerColor = "red";
+export const blueContainerColor: ContainerColor = "blue";
+export const yellowContainerColor: ContainerColor = "yellow";
+
+export type ContainerColor = "red" | "blue" | "yellow";
+
+export const FourInchContainerSize: ContainerSize = 4;
+export const SixInchContainerSize: ContainerSize = 6;
+export const EightInchContainerSize: ContainerSize = 8;
```

Comment:
I'll add some tests for this. I am using makeValidIdentifier here, so it would actually be:
```typescript
export const _4ContainerSize: ContainerSize = 4;
export const _6ContainerSize: ContainerSize = 6;
export const _8ContainerSize: ContainerSize = 8;
```

Reply:
Opened an issue to add validation for the identifiers #5502